test/test_inputs.py

```diff
@@ -8,7 +8,9 @@
 import os
 import tempfile
 import json
-import shutil
+
+
+os.environ["GRADIO_ANALYTICS_ENABLED"] = ""  # Disables analytics
```

Comment:
You overuse comments. If it's unclear that this line disables analytics (which I agree is unclear), then do something like `os.environ["GRADIO_ANALYTICS_ENABLED"] = "False"`

Reply:
That actually doesn't work. `bool("False") = True` so the standard way to set an env variable as `False` is by setting it to `""`